z2/docs/deposit.md

```diff
@@ -1,34 +1,49 @@
 # z2 deposit
 
-`z2 deposit` deposit the $ZILs to the deposit smart contract to promote a node as validator.
+The `z2 deposit` command deposits ZIL tokens to the deposit smart contract to promote a node as a validator.
+
 
 ```bash
-z2 deposit
+z2 deposit \
+  --chain <CHAIN_NAME> \
+  --public-key <BLS_PUBLIC_KEY> \
+  --peer-id <PEER_ID> \
+  --wallet <PRIVATE_KEY_OF_VALIDATOR> \
+  --amount <AMOUNT_IN_ZIL> \
+  --reward-address <REWARD_ADDRESS_OF_VALIDATOR>
 
-  --chain <CHAIN_NAME>
-  --public-key <PUBLIC_KEY>
-  --peer-id <PEER_ID>
-  --wallet <WALLET>
-  --amount <AMOUNT>
-  --reward-address <REWARD_ADDRESS>
 
-Usage: z2 deposit --chain <CHAIN_NAME> --public-key <PUBLIC_KEY> --peer-id <PEER_ID> --wallet <WALLET> --amount <AMOUNT> --reward-address <REWARD_ADDRESS>
+Usage: z2 deposit --chain <CHAIN_NAME> --public-key <BLS_PUBLIC_KEY> --peer-id <PEER_ID> --wallet <PRIVATE_KEY_OF_VALIDATOR> --amount <AMOUNT_IN_ZIL> --reward-address <REWARD_ADDRESS_OF_VALIDATOR>
```

Comment:
There are changes in #1226 that will probably impact this.

Reply:
Thank you for your observation, @shawn-zil. I will make sure to include you in my future PRs. For now, let's proceed with the current changes and make them live since we have an upcoming release. I will update the documentation again to include the new parameter we added. Thanks!